style.css

```diff
@@ -458,3 +473,10 @@ input {
     font-size: 1rem;
   }
 }
+
+@media screen and (min-width: 700px) and (max-width: 960px) {
+  .portfoilio-wrapper {
+    grid-template-columns: repeat(2, 1fr);
+    gap: 1rem;
+  }
```

Comment:
- [ ] Kindy make sure you use the same `typo` as the one used in the provided `Figma` template
![image](https://user-images.githubusercontent.com/97100091/192848627-9f1016ae-ef68-4e8a-b11e-b175f703b723.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/192848874-35387699-64ab-48cd-8bb5-a18699f78347.png)

Reply:
I kindly suggest following the directions of the previous reviewer, the original Figma design shows gray dots between the project descriptions